src/components/Shimmer/README.md

```diff
@@ -0,0 +1,9 @@
+# Shimmer
+
+## Usage
+
+```javascript
+import { Shimmer } from "KnitUI"
+
```

Comment:
Please create a table of props ( height, and other optional props that can be passed), also mention values that can be passed in props.